orchestration/hca_orchestration/solids/load_hca/utilities.py

```diff
@@ -1,20 +1,33 @@
 from typing import Optional
 
-from dagster import solid
+from dagster import solid, Failure
 from dagster.core.execution.context.compute import AbstractComputeExecutionContext
 from dagster_utils.contrib.data_repo.typing import JobId
 
+from hca_manage.check import CheckManager
 from hca_orchestration.contrib.slack import key_value_slack_blocks
+from hca_orchestration.models.hca_dataset import TdrDataset
 
 
 @solid(
-    required_resource_keys={'slack', 'target_hca_dataset', 'dagit_config'}
+    required_resource_keys={'slack', 'target_hca_dataset', 'dagit_config', 'data_repo_client'}
 )
 def terminal_solid(
         context: AbstractComputeExecutionContext,
         results1: list[Optional[JobId]],
         results2: list[Optional[JobId]]
 ) -> None:
+    target_hca_dataset: TdrDataset = context.resources.target_hca_dataset
+    check_result = CheckManager(
```

Comment:
This is running validation automatically after every import (unrelated to the BQ region updates)